lib/models/device-type.ts

```diff
@@ -16,8 +16,74 @@ limitations under the License.
 
 import type { InjectedDependenciesParam, PineOptions } from '..';
 import { DeviceType } from '../types/models';
+import { Contract } from '../types/contract';
 import { mergePineOptions } from '../util';
 import * as errors from 'balena-errors';
+import * as Handlebars from 'handlebars';
+import cloneDeep = require('lodash/cloneDeep');
+
+// REPLACE ONCE HOST OS CONTRACTS ARE GENERATED THROUGH YOCTO
+import { BalenaOS } from './balenaos-contract';
+
+const traversingCompile = (partials: any, initial: any, keys: string[]) => {
+	return Object.keys(partials).reduce(
+		(interpolated: any, partialKey) => {
```

Comment:
Can you type this better than `any` please

Reply:
Resolved in latest push